judge/models/tests/test_problem.py

```diff
@@ -245,6 +245,54 @@ def test_organization_admin_problem_methods(self):
         }
         self._test_object_methods_with_users(self.organization_admin_problem, data)
 
+    def give_basic_problem_ac(self, user):
+        Submission.objects.create(
+            user=user.profile,
+            problem=self.basic_problem,
+            result='AC',
+            points=self.basic_problem.points,
+            language=Language.get_python3(),
+        )
+
```

Comment:
If we merge this we should make sure we don't have any legacy problems left that abuse partial AC to hide submission sources instead of using the more recently-added submission visibility functionality.

Reply:
I think there's at most 2 problems that do this; shouldn't be an issue.